### PR TITLE
Assignment 2

### PR DESCRIPTION
**Pull request by abhishek-dash**

### CODE REVIEW

**Comment by saikiransathpadi** [comment]
correct, will look more good if you have used the border margins for all divisions

**Comment by abhishek-dash** [comment]
> correct, will look more good if you have used the border margins for all divisions

ok

File: src/index.html [hunk 1]
```diff
@@ -12,9 +12,81 @@
 	<div id="container">
 
 	<!-- Start your code from here -->
-	
-
-	<!-- End your code -->
+    <header>
+        <div class="head">
```
Comment (SaivijayK) [inline src/index.html:17]: background color is missing in header

File: src/index.html [hunk 2]
```diff
@@ -12,9 +12,81 @@
...
+            <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ8v6E1ZkaEouj18z_03BqvlmU1rEpnAJviKg&usqp=CAU" width="70px">
+        </div>
+        <div class="head fm">
+            <form>
+                <input type="search" name="find" placeholder='search'>
+            </form>
+        </div>
+		<nav class="navbar">
```
Comment (SaivijayK) [inline src/index.html:27]: background color is not needed in nav tag

File: src/index.html [hunk 3]
```diff
@@ -12,9 +12,81 @@
...
+            <ul>
+                <li><a  href="https://www.w3schools.in/c-tutorial/">Home</a></li>
+                <li><a  href="https://www.w3schools.in/c-tutorial/">About</a></li>
+                <li><a  href="https://www.w3schools.in/c-tutorial/">Services</a></li>
+                <li><a  href="https://www.w3schools.in/c-tutorial/">Contact Us</a></li>
+            </ul>
+        </nav>
+	</header>
+    <section class="sec">
```
Comment (SaivijayK) [inline src/index.html:37]: horizontal line is missing after header

File: src/index.html [hunk 4]
```diff
@@ -12,9 +12,81 @@
 	<div id="container">
```
Comment (SaivijayK) [inline src/index.html:12]: borders are missing for all the sections in code.

**Review by SaivijayK — COMMENTED** [review]
over all looks good, please go through reviews may looks better

**Pull request merged**